flight/PiOS/Common/pios_mpu9250_spi.c

```diff
@@ -539,7 +539,9 @@ void PIOS_MPU9250_SetGyroLPF(enum pios_mpu9250_gyro_filter filter)
  */
 void PIOS_MPU9250_SetAccelLPF(enum pios_mpu9250_accel_filter filter)
 {
-	PIOS_MPU9250_WriteReg(PIOS_MPU9250_ACCEL_DLPF_CFG_REG, filter);
```

Comment:
All these constants come from mpu60x0 except the mag, because that component of the chip did not change. i was making this constant match what is in the datasheet.